example.py

```diff
@@ -13,7 +13,7 @@
 from sqlalchemy.orm import scoped_session, sessionmaker
 from sqlalchemy.ext.declarative import declarative_base
 
-DATABASE_URI = 'sqlite:////tmp/github-flask.db'
+DATABASE_URI = 'sqlite:///tmp/github-flask.db'
```

Comment:
This is not wrong. You need 4 slashes to specify an absolute URL. See https://docs.sqlalchemy.org/en/13/dialects/sqlite.html#connect-strings